modules/express/src/lightning/lightningWalletRoutes.ts

```diff
@@ -36,3 +44,73 @@ export async function handleUpdateLightningWalletCoinSpecific(req: express.Reque
 
   return await lightningWallet.updateWalletCoinSpecific(params);
 }
+
+export async function handleListLightningTransactions(req: express.Request): Promise<unknown> {
+  const bitgo = req.bitgo;
+  const params = decodeOrElse(TransactionQuery.name, TransactionQuery, req.query, (error) => {
+    throw new ApiResponseError(`Invalid query parameters for listing lightning transactions: ${error}`, 400);
+  });
+
+  const coin = bitgo.coin(req.params.coin);
+  const wallet = await coin.wallets().get({ id: req.params.id });
+  const lightningWallet = getLightningWallet(wallet);
+
+  return await lightningWallet.listTransactions(params);
+}
+
+export async function handleGetLightningTransaction(req: express.Request): Promise<Transaction> {
+  const bitgo = req.bitgo;
+  const txId = req.params.txid;
+
+  if (!txId) {
+    throw new ApiResponseError('Missing required parameter: txid', 400);
+  }
```

Comment:
Use codecs here, do it as an immediate follow up please. 

1. list lightning payment
2. list lightning txn
3. list lightning invoice

Do it for these 3